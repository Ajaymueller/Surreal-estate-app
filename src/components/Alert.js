import React from 'react';
import PropTypes from "prop-types";

const Alert = ({ message, success }) => {
    <div className={`Alert alert-${success ? "success" : "error" }`}>
        {message}
    </div>

}

export default Alert;