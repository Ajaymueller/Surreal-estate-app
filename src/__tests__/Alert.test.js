import React from 'react';
import { render } from '@testing-library/react';
import Alert from '../components/Alert';
import { MemoryRouter } from 'react-router-dom';

const props = {
    message: "random message", 
    success: true, 
}

describe("alert", () => {
    it("component renders correctly", () => {
    const { asFragment } = render (
    <MemoryRouter><Alert { ...props } /></MemoryRouter> )
    expect(asFragment).toMatchSnapshot();
    });
});