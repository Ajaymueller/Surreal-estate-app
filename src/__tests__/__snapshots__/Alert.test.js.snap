// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`alert component renders correctly 1`] = `[Function]`;
